src/ledger/orderbook.ts

```diff
@@ -50,7 +51,11 @@ function formatBidsAndAsks(
   // for asks: lowest quality => lowest totalPrice/quantity => lowest price
   // for both bids and asks, lowest quality is closest to mid-market
   // we sort the orders so that earlier orders are closer to mid-market
-  const orders = _.sortBy(offers, 'quality').map(parseOrderbookOrder)
+
```

Comment:
(ultra-nit comment) I'd remove this newline to keep the comment closer to the code it's documenting